sklearn/linear_model/tests/test_ridge.py

```diff
@@ -195,14 +196,14 @@ def test_ridge_sample_weights():
             W = np.diag(sample_weight)
             if intercept is False:
                 X_aug = X
-                I = np.eye(n_features)
+                matrix = np.eye(n_features)
```

Comment:
matrix or array is potentially a numpy key word. So Let's use something else. Maybe `D` or are you getting errors for 1 char variables?